.github/workflows/ci.yaml

```diff
@@ -238,7 +235,7 @@ jobs:
   check-rust-docs:
     runs-on: ubuntu-24.04
     needs: changed_files
-    if: ${{needs.changed_files.outputs.rs_changed == 'true' || needs.changed_files.outputs.ci_changed == 'true' || needs.changed_files.outputs.windows_changed == 'true'}}
+    if: ${{needs.changed_files.outputs.rs_changed == 'true' || needs.changed_files.outputs.ci_changed == 'true'}}
```

Comment:
After reintroducing the windows_changed output, we'd need to undo this change, to rerun cargo-deny also on window-specific changes.